tests/test_upstream_issue.py

```diff
@@ -63,6 +63,29 @@ def setUp(self):
                 }
             }
         }
+        self.new_style_mock_github_message = {
+            'body': {
```

Comment:
Is `new_style_mock_github_message` supposed to be identical to `old_style_mock_github_message` except for the `body` vs. `msg` key?  If so, I recommend defining the body separately and using that definition to define these two attributes (or to define one attribute using the other), to avoid repeating ourself.

Reply:
You got it. Done in a305ae3.